### DISCUSSION TITLE
Miranda Availability

### DISCUSSION THREAD

**Discussion by Temsik-park** [bcholmes/StarTrek2d20#214] [General]
I noticed that the Miranda class isn't available at it's commission year, not being available until after TMP, despite Utopia Planetia placing it during the mid 2260's.

**Comment by bcholmes** [comment]
Good catch. I'll fix that.

**Comment by pawnnolonger** [comment]
Theres an issue with this. The stats in the book assume you're working from the refit date not the original launch date, this causes the Miranda math to be wrong if you start from the earlier date making it a slightly more powerful ship than its supposed to. I had this explained to me by the author who did the ship. The only way to fix it would to be to have 2 versions of the Miranda, pre refit, deducting 1 point from one of the systems, and post refit.
